tools/biosnoop_example.txt

```diff
@@ -4,33 +4,20 @@ Demonstrations of biosnoop, the Linux BPF/bpftrace version.
 This traces block I/O, and shows the issuing process (at least, the process
 that was on-CPU at the time of queue insert) and the latency of the I/O:
 
-# ./biosnoop.bt
 Attaching 4 probes...
-TIME(ms)     COMM             PID    LAT(ms)
-611          bash             4179        10
-611          cksum            4179         0
-627          cksum            4179        15
-641          cksum            4179        13
-644          cksum            4179         3
-658          cksum            4179        13
-673          cksum            4179        14
-686          cksum            4179        13
-701          cksum            4179        14
-710          cksum            4179         8
-717          cksum            4179         6
-728          cksum            4179        10
-735          cksum            4179         6
-751          cksum            4179        10
-758          cksum            4179        17
-783          cksum            4179        12
-796          cksum            4179        25
-802          cksum            4179        32
-[...]
+Start Time: 2020-07-31 16:03:09
+TIME(ms)     COMM             PID     DISK     OFFSET        LEN    LAT(ms)
+720          bash             1319718 nvme0n1  777994240     4096         0
+724          cksum            1321648 nvme0n1  680353792     16384        13
+725          cksum            1321648 nvme0n1  2021421056    4096         10
+725          cksum            1321648 nvme0n1  2021355520    4096         6
+725          cksum            1321648 nvme0n1  777986048     4096         10
+726          cksum            1321648 nvme0n1  1331789824    4096         15
```

Comment:
The LAT(ms) column does not match the format ("%7d").

Reply:
Fixed in the latest commit, I modified the times to keep the existing example sensible and apparently I am not a good printf parser :)